docs/display.rst

```diff
@@ -0,0 +1,69 @@
+Display
+*******
+
+.. py:module:: microbit.display
+
+This module controls the 5×5 LED display on the front of your board. It can
+be used to display images, animations and even text.
+
+Functions
+=========
+
+.. py:function:: get_pixel(x, y)
+
+    Return the brightness of the LED at column ``x`` and row ``y`` as an
+    integer between 0 (dark) and 9 (bright).
+
+
+.. py:function:: set_pixel(x, y, value)
+
+    Set the brightness of the LED at column ``x`` and row ``y`` to ``value``,
+    which has to be an integer between 0 and 9.
+
+
+.. py:function:: clear()
+
+    Set the brightness of all LEDs to 0.
+
+
+.. py:function:: print(string, delay=400)
+
+    Display the ``string`` one letter at a time, with ``delay`` milliseconds
+    between each pair of letters.
```

Comment:
On a second thought, it could just work with any iterable of images or characters.